yt_dlp/extractor/zee5.py

```diff
@@ -114,3 +114,45 @@ def _real_extract(self, url):
             'episode_number': int_or_none(try_get(json_data, lambda x: x['index'])),
             'tags': try_get(json_data, lambda x: x['tags'], list)
         }
+
+
+class Zee5SeriesIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://)(?:www\.)?zee5\.com/(?:global/)?(?:[^#/?]+/)?(?:(?:tvshows|kids)/[^#/?]+/)(?P<display_id>[^#/?]+)/(?P<id>[^#/?]+)/?(?:$|[?#])'
+    _TEST = {
```

Comment:
Add some more tests both here and in Zee5 that demonstrates the URLs with global/language. They can be added with `only_matching=True`. So no need of infodict